### ISSUE TITLE
no-trailing-spaces doesn't report trailing spaces after block comments

### ISSUE THREAD

**Issue by mdjermanovic** [eslint/eslint#12479]
**Tell us about your environment**

* **ESLint Version:**  6.5.1
* **Node Version:** 10.16.0
* **npm Version:** 6.9.0

**What parser (default, Babel-ESLint, etc.) are you using?**

default

**Please show your full configuration:**

<details>
<summary>Configuration</summary>

```js
module.exports = {
  parserOptions: {
    ecmaVersion: 2015,
  }
};
```

</details>

**What did you do? Please include the actual source code causing the issue, as well as the command that you used to run ESLint.**

[Online Demo Link](https://eslint.org/demo#eyJ0ZXh0IjoiLyogZXNsaW50IG5vLXRyYWlsaW5nLXNwYWNlczogW1wiZXJyb3JcIiwgeyBpZ25vcmVDb21tZW50czogdHJ1ZSB9XSovXG5cbi8qXG4gICAgVGhlcmUgYXJlIHRyYWlsaW5nIHNwYWNlcyBvbiB0aGUgbGluZSB3aGVyZSB0aGlzIGNvbW1lbnQgZW5kcy5cbiovICAgICAgICAgXG4iLCJvcHRpb25zIjp7InBhcnNlck9wdGlvbnMiOnsiZWNtYVZlcnNpb24iOjExLCJzb3VyY2VUeXBlIjoic2NyaXB0IiwiZWNtYUZlYXR1cmVzIjp7fX0sInJ1bGVzIjp7fSwiZW52Ijp7fX19)

```js
/* eslint no-trailing-spaces: ["error", { ignoreComments: true }]*/

/*
    There are trailing spaces on the line where this comment ends.
*/         

```

```bash
eslint index.js
```

**What did you expect to happen?**

1 error because the option ignores just spaces in comments, not those after comments?

**What actually happened? Please include the actual, raw output from ESLint.**

No errors.

**Are you willing to submit a pull request to fix this bug?**

Yes.

I'm not sure if this is intended behavior? I think there are no tests to confirm this.

**Comment by platinumazure** [comment]
Thanks @mdjermanovic for the issue!

This seems like a pretty clear bug to me. I assume this fix will increase warnings in some cases, so it should be an Update.

**Comment by mdjermanovic** [comment]
I'm working on this.
